packages/react/src/avatar-group/AvatarGroup.css.ts

```diff
@@ -0,0 +1,9 @@
+import { recipe } from "../vanilla-extract";
+
+export const avatarGroup = recipe({
+  base: [
+    {
+      display: "flex",
+    },
+  ],
+});
```

Comment:
ah i see now why we used this - to share a marker between avatar and avatar-group - let's do this a bit differently:

```suggestion
export const className = "ax-AvatarGroup";

export const avatarGroup = recipe({
  base: [
    className,
    {
      display: "flex",
    },
  ],
});
```

and in `Avatar.css.ts` we can re-use this `className` export

```ts
      selectors: {
        [`${styles.className} &:not(:first-child)`]: {
          marginLeft: "-12px",
        },
      },
```